doc/source/conf.py

```diff
@@ -180,7 +196,7 @@
 # The name for this set of Sphinx documents.
 # "<project> v<release> documentation" by default.
 #
-# html_title = u'pyarrow v0.1.0'
+# html_title = u'Apache Arrow v0.1.0'
```

Comment:
There is an unused `version` variable in the `conf.py` which We should set to `pyarrow.__version__` (suppose pyarrow is installed) and use here as `html_title = u'Apache Arrow {}'.format(version)`